test/battle/status1/burn.c

```diff
@@ -17,6 +17,23 @@ SINGLE_BATTLE_TEST("Burn deals 1/16th damage per turn")
             HP_BAR(player, damage: maxHP / 16);
     }
 }
+#else
+SINGLE_BATTLE_TEST("Burn deals 1/8th damage per turn")
```

Comment:
Instead of preproc'ing you could do and if else inside the first statement depending on the gen

Reply:
We could also just use a ternary for 16 : 8. That way, only the title needs to be checked

Reply:
resolved with ternary. good suggestion